benchmarks/hyperbahn-worker.js

```diff
@@ -107,10 +107,12 @@ HyperbahnWorker.prototype.createConfig = function createConfig() {
                 leafHost: '127.0.0.1',
                 leafPort: self.kafkaPort
             },
+            // 'clients.logtron.console': true,
```

Comment:
How does turning on console also turn on disk... Wat.

We can change the logFile to be /tmp